storage/CMakeLists.txt

```diff
@@ -662,6 +663,36 @@ target_link_libraries(quickstep_storage_SimpleScalarSeparateChainingHashTable
                       quickstep_utility_Alignment
                       quickstep_utility_Macros
                       quickstep_utility_PrimeNumber)
+target_link_libraries(quickstep_storage_SMAIndexSubBlock
```

Comment:
Similarly, `quickstep_storage_SMAIndexSubBlock` seems to need to move before `quickstep_storage_SplitRowStoreTupleStorageSubBlock`.